build.gradle

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 Aiven Oy
+ * Copyright 2021 Aiven Oy
```

Comment:
we do not need to change year. not critical. next time you do not need to do this

Reply:
Ok, decided to update since I touched it.